knimin/handlers/ag_pulldown.py

```diff
@@ -55,10 +63,19 @@ def post(self):
             blanks = self.get_argument('blanks').split(',')
         else:
             blanks = []
-        if self.get_argument('external'):
+
+        # query which surveys have been selected by the user
+        if self.get_argument('selected_ag_surveys', []):
```

Comment:
is this if really necessary? you could do:
selected_ag_surveys = map(int, self.get_argument('selected_ag_surveys', []).split(','))
right?